webapp/channels/src/selectors/calls.ts

```diff
@@ -32,7 +32,7 @@ export function getSessionsInCalls(state: GlobalState): Record<string, Record<st
 export function getCallsConfig(state: GlobalState): CallsConfig {
```

Comment:
Could you use `createSelector` for this since it's potentially returning a new object each time it's called?

Reply:
@hmhealey I assume the same should apply for `getSessionsInCalls` and `getCallsChannelState`?

Reply:
@hmhealey Actually, I didn't add that because it expects an arbitrary channel ID as input so I wasn't sure it made much sense.